keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -270,6 +264,14 @@ class ClusterHandler(
       }
         .mapNotNull { it.await() }
         .toSet()
+        .also { them ->
+          if (them.distinctBy { it.launchConfiguration.appVersion }.size == 1) {
+            publisher.publishEvent(ArtifactVersionDeployed(
+              resourceId = resource.id,
+              artifactVersion = them.first().launchConfiguration.appVersion
+            ))
```

Comment:
Publish an event if all the server groups in the cluster have the same appVersion

Reply:
oo good idea